app/validation/validator.py

```diff
@@ -1174,7 +1261,13 @@ def _get_answers_with_parent_ids(self, json_to_validate):
     def _get_questions_with_context(self, json):
         for section in json.get('sections'):
             for group in section.get('groups'):
-                for block in group.get('blocks'):
+                if group is None:
+                    print(section)
```

Comment:
As discussed, probably worth adding some more information here and returning an error, or using the logging library to allow setting level etc.

Reply:
I'm removing it from here and pushing it up into being called from much earlier parts of validation as it is not question-specific, and later validation methods break anyway when such basic problems exist in the schema.